### DISCUSSION TITLE
bbgo 用 github issue label 來標示懸賞

### DISCUSSION THREAD

**Discussion by yurenju** [lun-dao/LunDAO#53] [Ideas]
在 Twitter 上面看到 bbgo 直接用 [label 來標示 issue 懸賞][1]，覺得這個機制簡單好用，我們也可以以這個作為開頭，等到跑了一陣子之後再來用 smart contract 代替。
https://twitter.com/VagrantPi/status/1480813090262642689?s=20

![image](https://user-images.githubusercontent.com/22155/148909018-a54d105c-4870-47af-a503-15c3d1d54a78.png)

[1]: https://github.com/c9s/bbgo/issues

**Comment by hydai** [comment]
這樣也比較直覺，目前我們有打算提供哪些級別的 label 嗎？還是說讓使用者可以自己定義，雖然這樣會比較混亂一點
比如說：
1. 100
2. 300
3. 500
4. 1000
5. 更高？

**Reply by yurenju** [reply]
我是覺得讓使用者自己決定，然後我們來幫他加總，label 就用類似像這樣：
- `bounty: 100 LUN`
- `bounty: 100-500 LUN`
- `bounty: 500-1000 LUN`
- `bounty: 1000+ LUN`

我也開了 #55 來做這事

**Reply by hydai** [reply]
很可以，我記得有個發職缺的 repo 也是用 label 給年薪多少的 range 這樣
